### PR TITLE
Feature/loader

### PR DESCRIPTION
**Pull request by JeanVittory**
Building a hook to show a loader every time that a new pages is loaded

### CODE REVIEW

File: src/components/loading/Loading.components.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Spinner } from '@chakra-ui/react';
+import './loading.components.scss';
+
+export const Loading = () => {
+  return (
+    <div className="loading">
+      <Spinner />
+    </div>
+  );
+};
```
Comment (ht1204) [inline src/components/loading/Loading.components.jsx:10]: A quick refactor of this component could be the following, given that is just returning the component to implement in Virtual DOM:
```javascript
export const Loading = () =>  (
 <div className="loading">
    <Spinner />
 </div>
);
```

Comment (JeanVittory) [reply src/components/loading/Loading.components.jsx]: the suggested change was implemented. Look  the last commit in this branch

**Review by ht1204 — APPROVED** [review]
Awesome! Keep moving forward!

File: src/components/loading/Loading.components.jsx (hunk 1 above)
Comment (ht1204) [reply src/components/loading/Loading.components.jsx]: Great, keep rocking! 🚀

**Pull request merged**